modules/virtualization/hetzner-cloud.nix

```diff
@@ -1,6 +1,40 @@
-{ modulesPath, ... }:
-{
+{ config, lib, pkgs, modulesPath, ... }:
+
+with lib;
```

Comment:
I think I would rather this be specifically in the `options` section instead of poluting the highest scope since `lib.` is used in some places anyway

Reply:
I don't like `with` too, but this is just a part of standard nixos module structure, most nixpkgs modules are like this. `lib` is used in both `config` and `options` though, so I guess I'll just have explicit `lib.` everywhere to not bother with it.

Reply:
I've been rather fond of doing something akin to explicit imports, such as `inherit (lib) mkForce mkFoo mkBaz`. It serves as a sort of declaration of intent, along with making it explicit where something comes from.

Reply:
Yeah, finding which `with` a declaration comes from can be annoying